go.sum

```diff
@@ -1,17 +1,29 @@
-github.com/google/go-querystring v1.0.0 h1:Xkwi/a1rcvNg1PPYe5vI8GbeBY/jrVuDX5ASuANWTrk=
+github.com/beatgammit/turnpike v0.0.0-20170911161258-573f579df7ee/go.mod h1:nLl3qHMc5xKNLHHm/T7qBzrYGKSCJqLnFVLb2B5RvGI=
```

Comment:
Have you added any new packages that I missed? Otherwise, please revert these changes as well as with go.mod